parm/ocnicepost/ice_sfs.csv

```diff
@@ -0,0 +1,63 @@
+                'meltt_h',2,'Ct', 'bilinear' ,'','','', 'top ice melt' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'melts_h',2,'Ct', 'bilinear' ,'','','', 'top snow melt' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'meltb_h',2,'Ct', 'bilinear' ,'','','', 'basal ice melt' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'meltl_h',2,'Ct', 'bilinear' ,'','','', 'lateral ice melt' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'fresh_h',2,'Ct', 'bilinear' ,'','','', 'freshwtr flx ice to ocn (cpl)' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'fsalt_h',2,'Ct','conserve','','','', 'salt flux ice to ocn (cpl)' , 'kg/m^2/s' ,0,0,0,0,0,0,0,0
+                'fbot_h',2,'Ct','conserve','','','', 'heat flux ice to ocean (fbot)' , 'W/m^2' ,0,0,0,0,0,0,0,0
+                'fhocn_h',2,'Ct','conserve','','','', 'heat flux ice to ocn (cpl)' , 'W/m^2' ,0,0,0,0,0,0,0,0
+                'strairx_h',2,'Bu_x','conserve','strairy_h','Bu_y','', 'atm/ice stress (x)' , 'N/m^2' ,0,0,0,0,0,0,0,0
+                'strairy_h',2,'Bu_y','conserve','strairx_h','Bu_x','', 'atm/ice stress (y)' , 'N/m^2' ,0,0,0,0,0,0,0,0
+                'divu_h',2,'Ct', 'bilinear' ,'','','', 'strain rate (divergence)' , '%/day' ,0,0,0,0,0,0,0,0
+                'shear_h',2,'Ct', 'bilinear' ,'','','', 'strain rate (shear)' , '%/day' ,0,0,0,0,0,0,0,0
+                'dvidtt_h',2,'Ct', 'bilinear' ,'','','', 'volume tendency thermo' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'dvidtd_h',2,'Ct', 'bilinear' ,'','','', 'volume tendency dynamics' , 'cm/day' ,0,0,0,0,0,0,0,0
+                'daidtt_h',2,'Ct', 'bilinear' ,'','','', 'area tendency thermo' , '%/day' ,0,0,0,0,0,0,0,0
+                'daidtd_h',2,'Ct', 'bilinear' ,'','','', 'area tendency dynamics' , '%/day' ,0,0,0,0,0,0,0,0
+                'mlt_onset_h',2,'Ct', 'bilinear' ,'','','', 'melt onset date' , 'day of year' ,0,0,0,0,0,0,0,0
+                'frz_onset_h',2,'Ct', 'bilinear' ,'','','', 'freeze onset date' , 'day of year' ,0,0,0,0,0,0,0,0
+                'sitemptop_h',2,'Ct', 'bilinear' ,'','','', 'sea ice surface temperature' , 'K' ,0,0,0,0,0,0,0,0
+                'sitempsnic_h',2,'Ct', 'bilinear' ,'','','', 'snow ice interface temperature' , 'K' ,0,0,0,0,0,0,0,0
+                'sitempbot_h',2,'Ct', 'bilinear' ,'','','', 'sea ice bottom temperature' , 'K' ,0,0,0,0,0,0,0,0
+                'apond_h',2,'Ct', 'bilinear' ,'','','', 'melt pond fraction of sea ice' , '1' ,0,0,0,0,0,0,0,0
+                'hpond_h',2,'Ct', 'bilinear' ,'','','', 'mean melt pond depth over sea ice' , 'm' ,0,0,0,0,0,0,0,0
+                'ipond_h',2,'Ct', 'bilinear' ,'','','', 'mean pond ice thickness over sea ice' , 'm' ,0,0,0,0,0,0,0,0
+                'apeff_h',2,'Ct', 'bilinear' ,'','','', 'radiation-effective pond area fraction of sea ice' , '1' ,0,0,0,0,0,0,0,0
+                'uatm_h',2,'Bu_x', 'bilinear' ,'vatm_h','Bu_y','', 'atm velocity (x)' , 'm/s' ,0,0,0,0,0,0,0,0

```

Comment:
The atm winds (uatm_h, vatm_h) are also not on the Bu points. Again, please check a native CICE history file

```
uatm_1:coordinates = "TLON TLAT time" ;
```